apache/superset
# fix: Jumpiness of datasource modal

### SUMMARY

Partially fixes #12503


### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

BEFORE:

https://user-images.githubusercontent.com/60598000/104509562-210adb80-55ea-11eb-8344-d9d753eb0d89.mp4

AFTER:

https://user-images.githubusercontent.com/60598000/104509580-26682600-55ea-11eb-92f6-efe86d1192ac.mp4


### TEST PLAN
1. Open the datasource modal in Explore
2. Make sure the modal has a fixed height and jumpiness is removed

### ADDITIONAL INFORMATION


- [x] Has associated issue: #12503
